pkg/controller/migmigration/task.go

```diff
@@ -1362,6 +1379,66 @@ func (t *Task) hasImageStreams() (bool, error) {
 	return false, nil
 }
 
+// Get whether the associated plan has relevant artifacts using the analytics resource.
+func (t *Task) hasRelevantArtifacts() (bool, string, error) {
+	noMsg := ""
+	client, err := t.getDestinationClient()
+	if err != nil {
+		log.Trace(err)
+		return false, noMsg, err
+	}
+
+	analytic := migapi.MigAnalytic{}
+
+	err = client.Get(context.Background(),
+		k8sclient.ObjectKey{
+			Name:      t.PlanResources.MigPlan.Name,
```

Comment:
This `client.Get()` will fail if the user is doing a CLI driven migration. If I'm remembering correctly, MigAnalytics are only created when doing a UI driven migration.

Reply:
@djwhatle This PR was originally meant to check if there are any resources available in a namespace and if none is available, raise a warning. If the MigAnalytics is not suitable, would the method suggested in [comment](https://github.com/konveyor/mig-controller/pull/811#issuecomment-746547684 ) help?